shared-kcp/add_workload.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
+SCRIPT_DIR="$(cd "$(dirname "$0")" >/dev/null ; pwd)"
+KUBECONFIG_KCP="$SCRIPT_DIR/sharedkcp.kubeconfig"
+
+KUBECONFIG=$KUBECONFIG kubectl apply -k $SCRIPT_DIR
+
+KUBECONFIG=$KUBECONFIG kubectl config set-context --current --namespace=kcp-pcluster
+KUBECONFIG=$KUBECONFIG ./create-workload-cluster.sh kcp > workload-cluster.yaml
+
+KUBECONFIG=$KUBECONFIG_KCP kubectl apply -f workload-cluster.yaml
```

Comment:
You don't need to prefix every command by `KUBECONFIG=$KUBECONFIG`. Just export the env var and you'll be all set.

Reply:
I'm aware. I do this so that its clear for the user on where they are creating resources (physical or kcp). Plus its a small script anyways :)